linode/manager
# M3-5296: Add banner for Block Storage availability in Atlanta

## Description

Adds a banner to the Volumes Landing page and the Linode Details page if the Linode is in Atlanta that's hidden behind a feature flag. Dismiss both banners if either of them are dismissed. The Create Volume page will also include a message.

I included a fix for Kubernete's Upgrade Version banner since the font size wasn't being applied and updated the `Notice` component to match the styles used in the `DismissableBanner`.

## How to test

- Go to `/volumes` and check to see if the banner is visible depending on the flag.
- Go to `/volumes/create` and check to see if the notice is visible depending on the flag.
- Go to a Linode Details page and check to see if the banner is visible depending on the flag and if the Linode is located in Atlanta. 
  - The CTA button will also update based on if there's already a Volume created in Atlanta.
  - The drawer mode will change depending on what state it's in.
